client.coffee

```diff
@@ -0,0 +1,37 @@
+# ROCKAUTH
+
+class @rockauth
+
+  @url: (value) ->
+    @url.value = value.replace(/\/$/, '') if value
+    @url.value
+
+  @id: (value) ->
+    @id.value = value if value
+    @id.value
+
+  @secret: (value) ->
+    @secret.value = value if value
+    @secret.value
+
+  @authenticate: (opts = {}) ->
```

Comment:
Let's rename this to something like `authenticateWithPassword` (or something shorter if you can come up with a name), because in the future will allow authenticating in other ways to rockauth.